examples/SharedMemoryVideoWriter/main.cpp

```diff
@@ -0,0 +1,62 @@
+#include <pangolin/pangolin.h>
+
+#include <pangolin/utils/condition_variable.h>
+#include <pangolin/utils/shared_memory_buffer.h>
+#include <pangolin/utils/timer.h>
+
+#include <cmath>
+#include <cstdint>
+
+// This sample acts as a soft camera. It writes a pattern of GRAY8 pixels to the
+// shared memory space. It can be seen in Pangolin's SimpleVideo sample using
+// the shmem://example?size=640x480 video URI.
```

Comment:
Pangolin Video URI's are of the form scheme:[option1=value1,option2=value2,...]//device_name